sale_order_type/models/sale.py

```diff
@@ -16,12 +16,6 @@ class SaleOrder(models.Model):
         compute="_compute_sale_type_id",
         precompute=True,
         store=True,
-        readonly=False,
```

Comment:
Don't remove `readonly=False`. If you do this, you are saying to the ORM that the field is not computed writable.

Reply:
I already put it back